app/Http/Serializers/IpmaWarningSerializer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace VOSTPT\Http\Serializers;
+
+use Tobscure\JsonApi\AbstractSerializer;
+
+class IpmaWarningSerializer extends AbstractSerializer
+{
+    /**
+     * {@inheritDoc}
+     */
+    protected $type = 'ipma_warnings';
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getAttributes($model, array $fields = null): array
+    {
+        return [
+            'uuid'                => $model->uuid,
+            'text'                => $model->text,
```

Comment:
Can we get a better name, instead of `text`?

Reply:
description is ok for you ?

Reply:
That works